compiler/rustc_target/src/spec/mod.rs

```diff
@@ -1830,7 +1833,7 @@ impl TargetWarnings {
 #[derive(PartialEq, Clone, Debug)]
 pub struct Target {
     /// Target triple to pass to LLVM.
-    pub llvm_target: StaticCow<str>,
+    pub llvm_target: MaybeLazy<str>,
```

Comment:
What is the policy for converting `StaticCow`s to `MaybeLazy`s?
I see there are many `StaticCow`s remaining.

Only fields initialized to non-literals in built-in targets are converted?

Reply:
I have been fairly conservative about changing fields to use `MaybeLazy`, mainly because I didn't feel changing everything without first discussing about it but also because I don't think it's necessary.

The policy I followed was to change fields may that allocate memory or change their value based on some environment variable, like `llvm_name` (env access with apple targets) or the link args (`BTreeMap` allocation by many targets), ...

But many fields are static on all targets and don't change, so I don't see the benefits of changing them.